### ISSUE TITLE
[MAC] cannot import skimage 

### ISSUE THREAD

**Issue by xiuchengyang** [scikit-image/scikit-image#3559]
## Description
Hi,
I cannot import skimage in the python scripts. I have tried lots of ways and I still cannot fix the bug.
I utilized MAC OS system and python2.7. I installed skimage by pip install scikit-image.
Thanks for any advice.

[If reporting a bug, attach the entire traceback from Python.]
[If proposing an enhancement/new feature, provide links to related articles, reference examples, etc.]

## Way to reproduce
[If reporting a bug, please include the following important information:]
**>>> from skimage import segmentation**
/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/dask/array/numpy_compat.py:14: RuntimeWarning: divide by zero encountered in divide
  not np.allclose(np.divide(1, .5, dtype='i8'), 2) or
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/skimage/__init__.py", line 167, in <module>
    from .util.dtype import (img_as_float32,
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/skimage/util/__init__.py", line 6, in <module>
    from .apply_parallel import apply_parallel
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/skimage/util/apply_parallel.py", line 8, in <module>
    import dask.array as da
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/dask/array/__init__.py", line 5, in <module>
    from .core import (Array, block, concatenate, stack, from_array, store,
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/dask/array/core.py", line 31, in <module>
    from . import chunk
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/dask/array/chunk.py", line 68, in <module>
    nanprod = np.nanprod
AttributeError: 'module' object has no attribute 'nanprod'
**>>> import skimage**
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/skimage/__init__.py", line 167, in <module>
    from .util.dtype import (img_as_float32,
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/skimage/util/__init__.py", line 12, in <module>
    from ._montage import montage, montage2d
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/skimage/util/_montage.py", line 2, in <module>
    from .. import exposure
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/skimage/exposure/__init__.py", line 1, in <module>
    from .exposure import histogram, equalize_hist, \
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/skimage/exposure/exposure.py", line 4, in <module>
    from ..color import rgb2gray
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/skimage/color/__init__.py", line 1, in <module>
    from .colorconv import (convert_colorspace,
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/skimage/color/colorconv.py", line 60, in <module>
    from ..util import dtype, dtype_limits
ImportError: cannot import name dtype

**Comment by sciunto** [comment]
Please, be more specific on "a lot of ways". Can you describe what you did in details?

**Comment by jni** [comment]
Hi @xiuchengyang 

It looks like you tried to install scikit-image in your system Python install. (This is the Python software that your Mac's operating system uses for system tasks.) Generally this is considered bad practice these days, mainly because you can accidentally break your system Python install. Instead, I recommend you use Anaconda, which installs a separate Python as well as a whole bunch of scientific packages:

https://www.anaconda.com/download

Please let us know if this fixes your problem!

**Comment by xiuchengyang** [comment]
hi @jni @sciunto 
Thanks for your reply. After using Anaconda, it works fine now using the Conda interpreter. 
But for the python interpreter, the problem still exists. I am not sure whether there is a possible solution to it. As you said, the problem may come from the system tasks. 
(i) I first utilized "pip install scikit-image" and the installation went to the warning.
(ii) Then I utilized "pip install scikit-image --user" and everything looks fine. The package is located in the python site-packages file, and it is in the "pip list". However, if I import scikit-image, the above error information occurs.
Besides, I also tried to modify the .bash_profile which also did not work. Yet I can utilize some other packages, like openCV and GDAL.
I really appreciate your advice. And thanks again for your help.
